pages/index.js

```diff
@@ -1,59 +1,77 @@
-import { ContainerMain } from "@/components/styledComponents/Container.styled";
-import Link from "next/link";
 import { useState } from "react";
-import styled from "styled-components";
-import PageLanguage from "../components/language/languages";
-import PageLevel from "../components/level/level";
-import PageOptions from "../components/options/options";
-import { useEffect } from "react";
+import twisters from "/public/twisters";
 
 export default function HomePage() {
-  const [selectedLanguage, setSelectedLanguage] = useState("English");
-  const [selectedOptions, setSelectedOptions] = useState(
-    "Simple Sentences & For Kids"
-  );
-  const [selectedLevels, setSelectedLevels] = useState("Basic");
-  //const [frenchText, setFrenchText] = useState("");
-
-  function handleNext() {
-    /*  fetch("french.json")
-      .then((res) => res.json())
-      .then((data) => console.log(data));*/
-  }
+  const [config, setConfig] = useState({
+    language: "",
+    theme: "",
+    difficulty: "",
+  });
 
-  function handleOption(option) {
-    setSelectedOptions(option);
-    console.log("option est " + option);
-  }
-  function handLeLevel(level) {
-    setSelectedLevels(level);
-    console.log("  Level est" + level);
-  }
-
-  function handleLanguage(language) {
-    setSelectedLanguage(language);
-    console.log(" language est " + language);
-  }
+  const selectedTwister = twisters.find((twister) => {
+    return (
+      twister.language === config.language &&
+      twister.theme === config.theme &&
+      twister.difficulty === config.difficulty
+    );
+  });
 
   return (
     <>
-      <ContainerMain>
-        <h2> Homepage</h2>
-        <h2> Text Language </h2>
-        <PageLanguage handleLanguage={handleLanguage} />
-        <h2> Text Option</h2>
-        <PageOptions handleOption={handleOption} />
-        <h2> Text Level </h2>
-        <PageLevel handLeLevel={handLeLevel} />
-        <StyledLink href={"/folgePage"} onClick={handleNext}>
-          {" "}
-          Next
-        </StyledLink>
-      </ContainerMain>
+      <h2> tongue twister</h2>
+
+      <label htmlFor="lang-select">Language:</label>
+      <select
+        required
+        name="lang-select"
+        id="lang-select"
+        value={config.language}
+        onChange={(event) => {
+          setConfig({ ...config, language: event.target.value });
+        }}
+      >
+        <option value="">--Please choose an option--</option>
```

Comment:
You could switch this type of value into something like:
<option value="none" selected disabled hidden>Please choose an option</option>

I just did a quick google search, so it might not work in react, but i think that would make more sense to not display your information option to the user after they select something. The point is to just show this option at the initial option-pick and they disable/hide it afterwards.